app/models/comment.rb

```diff
@@ -0,0 +1,7 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+  def update_comments_counter
+    post.update_comments_counter
+  end
```

Comment:
- [ ] Kindly make sure that this function works as expected, I created a user(s), then I created a post on behalf of a user, there after I created comments(7) on that post, but the counter did not record anything. Kindly work on this issue and fix it.
__HINT:__
```Ruby
def update_comments_counter
  post.increment!(:comments_counter)
end
```


![counter](https://github.com/BANCUNGUYE66/Blog-app/assets/53124548/61c3c7fa-e366-47ba-89cb-f3b4c4a48ac5)